config/PropertyNames.xml

```diff
@@ -301,64 +301,64 @@ generated from the section label.
     <class name="mx" prefix-only="false">
         <suffix name="GroupBy" />
         <suffix name="Map" />
-        <suffix name="RetainDetached" />
+        <suffix name="RetainDetached" default="10" />
         <suffix name="Accept" />
         <suffix name="Reject" />
     </class>
 
     <section name="Ice">
-        <property name="AcceptClassCycles" />
+        <property name="AcceptClassCycles" default="0" />
         <property name="ACM.Client" deprecated="true"/>
         <property name="ACM.Server" deprecated="true"/>
         <property name="ACM" class="acm"/>
         <property name="ACM.Client" class="acm"/>
         <property name="ACM.Server" class="acm"/>
         <property name="Admin" class="objectadapter" />
-        <property name="Admin.DelayCreation" />
+        <property name="Admin.DelayCreation" default="0" />
         <property name="Admin.Enabled" />
         <property name="Admin.Facets" />
         <property name="Admin.InstanceName" />
-        <property name="Admin.Logger.KeepLogs" />
-        <property name="Admin.Logger.KeepTraces" />
+        <property name="Admin.Logger.KeepLogs" default="100" />
+        <property name="Admin.Logger.KeepTraces" default="100" />
         <property name="Admin.Logger.Properties" />
         <property name="Admin.ServerId" />
-        <property name="BackgroundLocatorCacheUpdates"/>
+        <property name="BackgroundLocatorCacheUpdates" default="0" />
         <property name="BatchAutoFlush" deprecated="true"/>
-        <property name="BatchAutoFlushSize" />
+        <property name="BatchAutoFlushSize" default="1024" />
         <property name="ChangeUser" />
-        <property name="ClassGraphDepthMax" />
+        <property name="ClassGraphDepthMax" default="100" />
         <property name="ClientAccessPolicyProtocol" />
-        <property name="Compression.Level" />
+        <property name="Compression.Level" default="1" />
         <property name="Config" />
         <property name="Connection" class="connection"/>
         <property name="ConsoleListener" />
```

Comment:
The default for ConsoleListener is undocumented. It's 1 in the code (C# only).

Reply:
I opened an issue to fix the docs